extension/intents/search/search.js

```diff
@@ -125,6 +125,59 @@ this.intents.search = (function() {
     }, CARD_POLL_INTERVAL);
   }
 
+  async function moveResult(context, step) {
+    stopCardPoll();
+    const { tabId, searchInfo } = await getSearchInfo();
+    if (!searchInfo) {
+      const e = new Error("No search made");
+      e.displayMessage = "You haven't made a search";
+      throw e;
+    }
+    if ((searchInfo.index >= searchInfo.searchResults.length - 1 && step > 0) ||
```

Comment:
Looks like there's some formatting issues here. Did you run `npm run format` ?

Reply:
Only eslint integration in Atom editor (0 errors) and npm install & npm start to test the code (following the README).Sorry I'll do npm run format before any commit.